controls/datagrid/columns/header.md

```diff
@@ -66,6 +66,31 @@ Define the `HeaderContentTemplate` in the DataGridColumn:
 
 ![.NET MAUI DataGrid Column Header Template](../images/header-content-template.png)
 
+## Hover Visual State on Column Header
+
+You can change the background color of the column header on hover visual state by setting the `BackgroundColor` property.
+
+This example demonstrates how to apply `BackGroundColor` to the DataGrid Header on Hover Visual State: 
+
+```XAML
+ <VisualStateManager.VisualStateGroups>
```

Comment:
Create sdk example for this case or include the approach in existing example inside the styling section. then add the code snippet to the documentation.